config/config.go

```diff
@@ -201,6 +204,10 @@ func UnregisterProviders() {
 // Load creates a Provider for use in a service
 func Load() Provider {
 	var static []Provider
+
+	if len(_configFiles) == 0 {
```

Comment:
I don't like in init, but I can set it as a default values in variable and skip this step.